src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStringToArray = sourceString
```

Comment:
There is a critical issue here: the division operator `/` is used instead of the assignment operator `=`. This will cause a syntax error. You should replace `/` with `=` to assign the result of `sourceString.split(';')` to `sourceStringToArray`.